docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,57 @@
+# How to upgrade bootstrap Kubelet
+
+## Contents
+
+- [Introduction](#introduction)
+- [Steps](#steps)
+  - [Drain the node](#drain-the-node)
+  - [Find out the IP and SSH](#find-out-the-ip-and-ssh)
+  - [On the node](#on-the-node)
+- [Caveats](#caveats)
+
+## Introduction
+
+Bootstrap Kubelet is the Kubelet running on the node as a systemd service outside Kubernetes cluster's control. Unlike most cluster components, `lokoctl` cannot update it(which may change in the future). This document enlists the manual steps involved in updating the bootstrap Kubelet.
```

Comment:
I'd add line breaks here and add more explanation about the concept of self hosted kubelet.

```suggestion
Lokomotive cluster runs 2 different sets of Kubelet processes. Initially, **bootstrap** kubelet configured on the node as a rkt container performs joining the cluster and then Kubelet pod managed using DaemonSet takes over the node one, which allows to to perform seamless updates between Kubernetes patch version for nodes and also to tune nodes configuration using tools like `kubectl`.

At the time of writing `lokoctl` cannot update node Kubelet, so this document explains how to perform this operation manually. However, if your cluster has self-hosted Kubelet disabled, this guide is not relevant.
```

Reply:
> However, if your cluster has self-hosted Kubelet disabled, this guide is not relevant.

Self hosted kubelet of or not this doc is relevant to everyone.

Reply:
The line breaks and explanation is still valuable to be included IMO.

Reply:
done